src/components/Mypage/AutoLogin.js

```diff
@@ -0,0 +1,26 @@
+import React, { useCallback, useState } from 'react';
+import { getCookie, setCookie } from '../Shared/Cookies';
+import * as S from './styles';
+
+const AutoLogin = () => {
+  const isAutoLogedIn = getCookie('isAuto');

```

Comment:
이런 오타 계속해서 나오는 것 같아요. 

```suggestion
  const isAutoLoggedIn = getCookie('isAuto');

```

Reply:
ebc2194